### PR TITLE
Docs: remove "develop" branch from the `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by nickyito**
## Overview

This PR removes all the explanations that say "develop" branch from the `CONTRIBUTING.md` since the `develop` branch is not used in this project.

## Assignee Checklist:

- [x] The base branch is correct
- [x] The correct assignees and reviewers have been designated for this PR
- [x] All the related issues are associated with this PR
- [x] All criteria in the associated issues are met (please tick the checkboxes)
- [x] My changes do not generate new warnings or errors (especially in the console)

## Reviewer Checklist:

- [x] The code follows the generic best practices
- [x] Code is well-commented and easy to understand

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ If you are interested in the detailed specification you can visit:
 
 ### For Team Members:
 
-1. Create a new branch from the `develop` branch, or from the `main` branch if you're working on a hotfix.
+1. Create a new branch from the `man` branch.
```
Comment (hitohata) [inline CONTRIBUTING.md]: This looks typo.

**Review by hitohata — CHANGES_REQUESTED** [review]
Please fix a typo.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nickyito) [reply CONTRIBUTING.md]: This is on purpose! But if you don't like it, I'm willing to fix it 😆

**Review by hitohata — APPROVED** [review]
Thank you.

**Comment by nickyito** [comment]
@hitohata

Thank you for your review! I have fixed the point you pointed out.

**Pull request merged**